### PR TITLE
Improve NEWS wording.

### PR DESCRIPTION
**Pull request by cbrueffer**
The entry about Bio.Data is hard to parse; too me a while to understand that it's actually about several different datasets.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -32,8 +32,9 @@ When a gap character is defined, Bio.Seq will now translate gap codons
 is inferred from the Seq object's alphabet, but it can also be passed as an
 argument to the translate method.
 
-The new NCBI genetic code table 25, Candidate Division SR1 and Gracilibacteria,
-has been added to Bio.Data (and the translation functionality).
+Three new datasets have been added to Bio.Data (and the translation
```
Comment (peterjc) [inline NEWS]: Its one dataset (table 25), covering both Candidate Division SR1 and Gracilibacteria:

https://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi#SG25

File: NEWS [hunk 2]
```diff
@@ -17,14 +17,14 @@ support for Python 2.6 is considered to be deprecated. It has also been
...
-comparison (are they same instance in memory?). This can be surprising, but
+comparison (are they the same instance in memory?). This can be surprising, but
 comparing all of the attributes would be too complex. As of this release
 attempting to compare SeqRecord objects should raise an exception instead. If
 you want the old behaviour, use id(record1) == id(record2) instead.
 
 Following the convention used elsewhere in Biopython, there is a new function
 Bio.KEGG.read(...) for parsing KEGG files expected to contain a single record
-only - existing function Bio.KEGG.parse(...) is intended to be used to iterate
+only - the existing function Bio.KEGG.parse(...) is intended to be used to iterate
```
Comment (peterjc) [inline NEWS]: Adding "the" is fine, but line wrap before iterate (we try to keep to 78 chars in this file for old school terminal usage).

File: NEWS [hunk 3]
```diff
@@ -17,14 +17,14 @@ support for Python 2.6 is considered to be deprecated. It has also been
 tested on PyPy 2.4 to 2.6, PyPy3 version 2.4, and Jython 2.7.
 
 Comparison of SeqRecord objects until now has used the default Python object
-comparison (are they same instance in memory?). This can be surprising, but
+comparison (are they the same instance in memory?). This can be surprising, but
```
Comment (peterjc) [inline NEWS]: Good catch.

**Comment by cbrueffer** [comment]
Ah! Well, that just proves my point :-P

How about "The new NCBI genetic code table 25, covering Candidate Division SR1 and Gracilibacteria,...", that would already help a lot.

**Comment by peterjc** [comment]
Indeed, clearly it was open to confusion. I like your new proposed wording.

**Comment by cbrueffer** [comment]
I have updated the branch to take all suggestions into account.

**Comment by peterjc** [comment]
Cherry picked as https://github.com/biopython/biopython/commit/450ea2551dac32091c80c4e9d0d63b98892aa122 to keep the main history simple (also fixed filename typo in commit comment NOTES -> NEWS). Thank you!
